.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         - 
           qt_version: "5.15.2"
           modules: ""
-          os: ubuntu-18.04
+          os: ubuntu-latest
```

Comment:
You can still use` os: ubuntu-20.04`. This seems to work for my branch and the generated AppImages work my Debian distro.

Reply:
I switched back to 18.04 due to #77 needing an older version of GCC for Qt5. Unless there is a reason for 20.04 then I'm fine using `ubuntu-latest`.